hack/minikube.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+set -e
+set -u
+GIT_COMMIT_ID= git rev-parse --short HEAD
+
+#set namespace here
+kubectl config set-context --current --namespace=default
+
+kubectl create -f deploy/role.yaml
+kubectl create -f deploy/role_binding.yaml
+kubectl create -f deploy/service_account.yaml
+kubectl create -f deploy/crds/operators.coreos.com_servicebindings_crd.yaml
+
+kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Might as well store in these yamls in a file so that they can used outside this script too ? :)